src/ontology/mondo-edit.obo

```diff
@@ -402073,8 +402076,13 @@ xref: OMIM:118450 {source="Orphanet:261619", source="Orphanet:261619/ntbt", sour
 xref: Orphanet:261619 {source="MONDO:equivalentTo"}
 xref: UMLS:C1956125 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:365434"}
 is_a: MONDO:0007318 {source="OMIM:118450", source="Orphanet:261619"} ! Alagille syndrome
-relationship: has_characteristic SO:1000008 ! point_mutation
+intersection_of: MONDO:0007318 ! Alagille syndrome
+intersection_of: has_characteristic HP:0000006 ! Autosomal dominant inheritance
```

Comment:
@katiermullen should the autosomal dominance inheritance be part of the definition (ie is it a required condition, because if it is recessive, it is another disease)? 
If it is, then this should be added to the textual definition. 
If it is not (and it is a "feature" of the disease), then it should be removed from the logical definition and added as a SC (which is already is)

Reply:
@katiermullen I think it should be removed from the logical definition